Use openstack-monitoring as the default namespace (#344)
infrawatch/documentation: doc-Service-Telemetry-Framework/assemblies/assembly_upgrading-service-telemetry-framework-to-version-1-3.adoc

[role="_abstract"]
To migrate from {Project} ({ProjectShort}) v1.2 to {ProjectShort} v1.3, you must replace the `ClusterServiceVersion` and `Subscription` objects in the `service-telemetry` namespace on your {OpenShift} ({OpenShiftShort}) environment.

> NOTE: The default namespace in {ProjectShort} v1.3 and earlier used `service-telemetry`, but has changed to `openstack-monitoring` in subsequent releases. {ProjectShort} can be installed to any namespace, and the existing namespace `service-telemetry` has been maintained for the purposes of upgrade documentation.

Reviewer: ```suggestion
NOTE: The default namespace in {ProjectShort} v1.3 and earlier used `service-telemetry`, but has changed to `openstack-monitoring` in subsequent releases. You can install {ProjectShort} to any namespace, and the existing namespace `service-telemetry` is maintained for the purposes of upgrade documentation.
```
